hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/placement/algorithms/ContainerPlacementPolicyFactory.java

```diff
@@ -36,7 +36,7 @@
 
   private static final Class<? extends ContainerPlacementPolicy>
       OZONE_SCM_CONTAINER_PLACEMENT_IMPL_DEFAULT =
-      SCMContainerPlacementRandom.class;
+      SCMContainerPlacementRackAware.class;
```

Comment:
Since we have ozone.network.topology.aware.read false by default, do we still need the placement policy to be SCMContainerPlacementRackAware by default?

Reply:
ContainerPlacementPolicy chosen  and topology aware read are two independent choice. If  SCMContainerPlacementRandom is used, we can still enable topology aware read as long as datanode to network location mapping is correctly built.  
This change is committed unintentionally, I will remove it now, and enable the configuration after 0.4.1 is released.

Reply:
Thanks for the details.